source/_posts/Creating-obfuscated-code-in-python.md

```diff
@@ -0,0 +1,58 @@
+
+---
+title: Creating obfuscated code in python 
```

Comment:
Title   tag character should be 50-60 character

Reply:
done, kindly check